services/QuillLMS/client/app/bundles/Evidence/libs/conceptResults.ts

```diff
@@ -16,25 +16,59 @@ export const generateConceptResults = (currentActivity, submittedResponses) => {
     so: 3
   }
 
+  // This list is manually copied from the production database.
+  // It is a very brittle integration since there's nothing that keeps it
+  // synced with what's actually going on in the database
+  const conjunctionToSemanticConceptUid = {
+    because: "qkjnIjFfXdTuKO7FgPzsIg",
```

Comment:
what about adding a `conjunctionToSemanticConceptUid` parameter to `generateConceptResults`?  In then populating those uid higher upstream via some initializer in Rails?

Reply:
I've refactored this per our Zoom discussion so that it now uses an API to fetch topic-optimal data for calculating concept results.